### PR TITLE
Created SolutionByKuljeet.cpp

### PR DESCRIPTION
**Pull request by Kuljeet-123**

### CODE REVIEW

File: Basic/Calculate Nth Fibonacci Number/SolutionByKuljeet.cpp [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+using namespace std;
+
+int fibonacci(int n)
+{
+    if(n <= 1)
+      return n;
+
```
Comment (shoaibrayeen) [inline Basic/Calculate Nth Fibonacci Number/SolutionByKuljeet.cpp:8]: if n == 0 - there's no 0th term
if n == 1, it's 0
if n == 2, it's 1

The series is 0,1,1,2,3,5,8...

Comment (shoaibrayeen) [reply Basic/Calculate Nth Fibonacci Number/SolutionByKuljeet.cpp:8]: Kindly modify!

**Comment by Kuljeet-123** [comment]
@shoaibrayeen  I have Updated the please review it

File: Basic/Calculate Nth Fibonacci Number/SolutionByKuljeet.cpp [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        return fibonacci(n-2) + fibonacci(n-1);
+}
+
+
+int main()
+{
+    int n = 0;
+    cout << "Enter value of n:" << endl;
+    cin >> n;
+    if(fibonacci(n) == -1)
```
Comment (shoaibrayeen) [inline Basic/Calculate Nth Fibonacci Number/SolutionByKuljeet.cpp:18]: You don't need to call the method if n is less than or equal to 0. Put a comment that Fibonacci Number doesn't exist for given position.

**Comment by Kuljeet-123** [comment]
@shoaibrayeen I have updated the code..  can you review it now

**Pull request merged**